SECURITY.md

```diff
@@ -0,0 +1,76 @@
+
+### Public Disclosure
+
+A public disclosure of security vulnerabilities is released alongside release updates or details that fix the vulnerability. We try to fully disclose vulnerabilities once a mitigation strategy is available. Our goal is to perform a release and public disclosure quickly and in a timetable that works well for users. For example, a release may be ready on a Friday but for the sake of users may be delayed to a Monday.
+
+CVEs will be assigned to vulnerabilities. Due to the process and time it takes to obtain a CVE ID, disclosures will happen first. Once the disclosure is public the process will begin to obtain a CVE ID. Once the ID has been assigned the disclosure will be updated.
+
+If the vulnerability reporter would like their name and details shared as part of the disclosure process we are happy to. We will ask permission and for the way the reporter would like to be identified. We appreciate vulnerability reports and would like to credit reporters if they would like the credit.
+
+## Security Team Membership
+
+The security team is made up of a subset of the OSM project maintainers who are willing and able to respond to vulnerability reports.
+
+### Responsibilities
+
+* Members MUST be active project maintainers on active (non-deprecated) OSM projects as defined in [OSM Contributor Ladder](/CONTRIBUTOR_LADDER.md)
+* Members SHOULD engage in each reported vulnerability, at a minimum to make sure it is being handled
+* Members MUST keep the vulnerability details private and only share on a need to know basis
+
+### Membership
+
+New members are required to be active maintainers of OSM projects who are willing to perform the responsibilities outlined above. The security team is a subset of the maintainers. Members can step down at any time and may join at any time.
+
+From time to time, OSM projects are deprecated. If at any time a security team member is found to be no longer be an active maintainer on active OSM projects, this individual will be removed from the security team.
+
+## Patch and Release Team
+
+When a vulnerability comes in and is acknowledged, a team - including maintainers of the OSM project affected - will assembled to patch the vulnerability, release an update, and publish the vulnerability disclosure. This may expand beyond the security team as needed but will stay within the pool of OSM project maintainers.
+
+## Disclosures
+
+Vulnerability disclosures are published as blog posts on the [OSM Blog](https://openservicemesh.io/blog/) and emailed to the [OSM mailing list](https://lists.cncf.io/g/cncf-openservicemesh-maintainers). The disclosures will contain an overview, details about the vulnerability, a fix for the vulnerability that will typically be an update, and optionally a workaround if one is available.
```

Comment:
Why should this be published as a blog? Isn't a security disclosure on the mailing list enough?

Reply:
@shashankram this is standard practice for most projects. Some users may not belong to the mailing list. The blog is just another way to communicate to users.